pkg/kubelet/images/image_manager.go

```diff
@@ -219,3 +317,58 @@ func applyDefaultImageTag(image string) (string, error) {
 	}
 	return image, nil
 }
+
+// isEnsuredBySecret returns two boolean values: pulledBySecret, ensuredBySecret.
+// 1. pulledBySecret return true is it is already in the ensured map, and it will
+// be removed from the ensured map once it is expired.
+// 2. ensuredBySecret returns true if the secret for an auth used to pull an
+// image has already been authenticated through a successful pull request
+// and the same auth exists for this podSandbox/image/.
```

Comment:
I'm not sure you mean to keep the second `/` in `podSandbox/image/`

Reply:
Updated.